k8s/Makefile

```diff
@@ -57,13 +23,16 @@ create-helm-charts:
 		> ${HELM_SERVERS_BASE}/.seldon-v2-servers.yaml
 	mv ${HELM_SERVERS_BASE}/.seldon-v2-servers.yaml ${HELM_SERVERS_BASE}/seldon-v2-servers.yaml
 
+.PHONY: create-yaml
+create-yaml:
+	helm template seldon-core-v2-certs ./helm-charts/seldon-core-v2-certs > yaml/certs.yaml
+	helm template seldon-core-v2-crds ./helm-charts/seldon-core-v2-crds > yaml/crds.yaml
+	helm template seldon-core-v2-components ./helm-charts/seldon-core-v2-setup > yaml/components.yaml
+	helm template seldon-core-v2-servers ./helm-charts/seldon-core-v2-servers > yaml/servers.yaml
+
 .PHONY: set-chart-version
 set-chart-version:
-	sed -i "s/version: .*/version: ${NEW_VERSION}/" helm-charts/seldon-core-v2-certs/Chart.yaml
-	sed -i "s/appVersion: .*/appVersion: ${NEW_VERSION}/" helm-charts/seldon-core-v2-certs/Chart.yaml
-	sed -i "s/version: .*/version: ${NEW_VERSION}/" helm-charts/seldon-core-v2-crds/Chart.yaml
-	sed -i "s/appVersion: .*/appVersion: ${NEW_VERSION}/" helm-charts/seldon-core-v2-crds/Chart.yaml
-	sed -i "s/version: .*/version: ${NEW_VERSION}/" helm-charts/seldon-core-v2-setup/Chart.yaml
-	sed -i "s/appVersion: .*/appVersion: ${NEW_VERSION}/" helm-charts/seldon-core-v2-setup/Chart.yaml
-	sed -i "s/version: .*/version: ${NEW_VERSION}/" helm-charts/seldon-core-v2-servers/Chart.yaml
-	sed -i "s/appVersion: .*/appVersion: ${NEW_VERSION}/" helm-charts/seldon-core-v2-servers/Chart.yaml
+	sed -i 's/\(version\|appVersion\): .*/\1: ${NEW_VERSION}/' helm-charts/seldon-core-v2-certs/Chart.yaml
```

Comment:
:information_source: For some reason, Make doesn't like use the extended-regex form for sed, at least on my machine.  When using that (via `-E` or `-r`), it complains about an invalid reference, but when running directly it's fine for me!  Perhaps a `sh` vs. `bash` difference?

Reply:
Fixed in https://github.com/SeldonIO/seldon-core/pull/4843/commits/01b26c13d30c2eadff04f29d0eaf97ab6d1481e3 - apparently needed to provide `-i -r` instead of `-ir`